awx/api/authentication.py

```diff
@@ -137,3 +137,12 @@ def authenticate_header(self, request):
         if not settings.AUTH_BASIC_ENABLED:
             return
         return super(LoggedBasicAuthentication, self).authenticate_header(request)
+
+
+class SessionAuthentication(authentication.SessionAuthentication):
+
+    def authenticate_header(self, request):
+        return 'Session'
+
+    def enforce_csrf(self, request):
```

Comment:
This function is override to be a no-op because it is meant to enforce more strict CSRF protection, API browser enforces that but UI does not. It may probably be removed depending on UI implementation.